### PR TITLE
media-sound/flacon: verbump to 4.1.0

### PR DESCRIPTION
**Pull request by Coacher**
Package-Manager: Portage-2.3.31, Repoman-2.3.9

### CODE REVIEW

File: media-sound/flacon/flacon-4.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+# Ignore rudimentary et, uz@Latn, zh_TW translation(s).
+PLOCALES="cs cs_CZ de el es es_MX fr gl hu it ja_JP lt ms_MY nb nl nl_BE pl pl_PL pt_BR pt_PT ro_RO ru sr sr@latin tr uk zh_CN"
+# Tests require lots of disk space
+CHECKREQS_DISK_BUILD=10G
+
+inherit check-reqs cmake-utils eutils gnome2-utils l10n virtualx xdg-utils
```
Comment (Zlogene) [inline media-sound/flacon/flacon-4.1.0.ebuild]: why is `eutils` here?

Comment (Coacher) [reply media-sound/flacon/flacon-4.1.0.ebuild]: It provides `optfeature`.

Comment (Zlogene) [reply media-sound/flacon/flacon-4.1.0.ebuild]: Indeed I overlooked it

**Comment by Coacher** [comment]
Thank you, @monsieurp.

**Pull request closed without merging**